The CLI Tool doesn't compile

### Describe the bug

The `theseus-cli` crate doesn't compile because the *optional* dependency `tauri` for `theseus` isn't optional:

https://github.com/modrinth/theseus/blob/3ff0ff238a4360960a8fee26d45094f92c8fefc5/theseus/Cargo.toml#L44

https://github.com/modrinth/theseus/blob/3ff0ff238a4360960a8fee26d45094f92c8fefc5/theseus/src/util/io.rs#L5-L8

Which has been changed in https://github.com/modrinth/theseus/commit/0d3f007dd4ab5b2fb6dadb09f073c28095b33e33

The compile error I got was:
```
error[E0433]: failed to resolve: use of undeclared crate or module `tauri`
 --> theseus/src/util/io.rs:7:5
  |
7 | use tauri::async_runtime::spawn_blocking;
  |     ^^^^^ use of undeclared crate or module `tauri`

For more information about this error, try `rustc --explain E0433`.
```

### Steps to reproduce

I tried to install the CLI by running
```
cargo install --git https://github.com/modrinth/theseus/ theseus_cli
```

### Expected behavior

The CLI should compile

### System information

N/A

### Additional context

Is the CLI even supported anymore? The change was done over a month ago and nobody noticed?

What is the recommended way to manage the modpack of a headless minecraft server? (Would that even be possible with the CLI?)

--- a/theseus/src/util/io.rs
+++ b/theseus/src/util/io.rs
@@ -4,7 +4,7 @@
 use std::{path::Path, io::Write};
 
 use tempfile::NamedTempFile;
-use tauri::async_runtime::spawn_blocking;
+use tokio::task::spawn_blocking;
 
 #[derive(Debug, thiserror::Error)]
 pub enum IOError {

